manifests/chatbot/base/bot.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: chatbot
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: chatbot-rbac
+subjects:
+  - kind: ServiceAccount
+    name: chatbot
+    namespace: default
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
note: these are quite broad permissions but 👌 for me

Reply:
yeah, I wanted to create a custom role, but I think it's not worth the effort and complexity.